index.html

```diff
@@ -246,7 +246,28 @@ <h2 class="media_header">LET'S CONNECT</h2>
         </div>
       </div>
     </section>
+
     </div>
+    <footer class="contact_container">
+      <div>
+        <div class="container flex">
+          <h1 class="feedback">contact me</h1>
+          <p class="msg">if you have an application you are interested in developing,
+            a feature that you need built or a project that needs coding. 
+            I'd love to help with it</p>
+        </div>
+        <div>
+          <form action="#" method="post">
+            <input type="text" aria-required="true" maxlength="30" name="name" placeholder="patrick rejoice">
+            <input type="email" aria-required="true" maxlength="30" name="email" placeholder="rmjspecial2015@gmail">
+            <textarea aria-required="true" maxlength="30" name="name" placeholder="Write your message here"></textarea>
```

Comment:
- [ ] Please make the max length 500. It should look something like this `maxlength="500"` 🏄

Reply:
Kindly set the maxlength property of the text area to 500 rather than 300. Also, the name of this field should be `message`